mailchimp_auth/views.py

```diff
@@ -54,40 +54,41 @@ class SignUpForm(JSONFormResponseMixin, FormView):
     def form_valid(self, form):
         email = form.cleaned_data['email']
 
-        token = form.data['g-recaptcha-response']
+        token = form.data.get('g-recaptcha-response', False)
```

Comment:
this can be 

```python
         token = form.data.get('g-recaptcha-response')
```

which will return a None if key is empty.

`if None` will be the same as `if False`